Issue #98 Known issues of AdGuard for Windows (#116)
AdguardTeam/KnowledgeBase: docs/adguard-for-windows/solving-problems/known-issues.md

---
title: Known issues of AdGuard for Windows
sidebar_position: 10
---

## AdGuard for Windows and AdGuard VPN for Windows compatibility issues

Once you install AdGuard for Windows and AdGuard VPN for Windows, they just start working together without any effort on your part. However, changing some default settings may prevent the apps from working correctly when running simultaneously.

> There are two specific settings in AdGuard for Windows: *Use redirect driver mode* and *Filter localhost*. By default, the *Use redirect driver mode* setting is disabled and the *Filter localhost* – is enabled. Changing the state of any of these settings will inevitably break AdGuard's filtering if AdGuard Ad Blocker and AdGuard VPN are enabled on your device.  

Reviewer: ```suggestion
There are two specific settings in AdGuard for Windows: *Use redirect driver mode* and *Filter localhost*. By default, the first one is disabled and the latter one is enabled. Changing any of these settings will inevitably disrupt AdGuard's filtering if both AdGuard Ad Blocker and AdGuard VPN are enabled on your device.  
```

> > Usually changing these settings default state is required to overcome issues related to the simultaneous operation of AdGuard Ad Blocker and various apps working with the network (e.g. antiviruses, VPNs, network filters, etc).

Reviewer 2: тут не только Ad Blocker

Reviewer 2: ну тут как бы возникает вопрос - мы пишем, что менять нельзя, но иногда изменения необходимы для компатибилити. С одной стороны ничего не добавить, но с другой стороны можно дополнительно указать, что мол такие редкие случаи мы будем рады рассмотреть отдельно, пишите таски (ссылка на гх).

> We are currently working on a solution to this problem.

Reviewer 2: `solution to this problem.` - немного относится к предыдущему комментарию, давай напишем это в формате выше

> Usually changing these settings default state is required to overcome issues related to the simultaneous operation of AdGuard Ad Blocker and various apps working with the network (e.g. antiviruses, VPNs, network filters, etc). If you come across a situation where you need AdGuard Ad Blocker and AdGuard VPN to work simultaneously and correctly, and the default state of the above settings needs to be changed — [create an issue on GitHub](https://github.com/AdguardTeam/AdguardForWindows/issues/new/choose) so we can help you privately.

Reviewer: ```suggestion
Changing these settings is only required to overcome issues related to the simultaneous operation of AdGuard Ad Blocker and apps working on the network level (e.g. antiviruses, VPNs, network filters, etc). If you come across a situation where you need AdGuard Ad Blocker and AdGuard VPN to work simultaneously and correctly, and the default state of the above settings needs to be changed — [create an issue on GitHub](https://github.com/AdguardTeam/AdguardForWindows/issues/new/choose) so we can help you personally.
```

> We are currently working towards overcoming the listrd above limitations of simultaneous work of our applications. 

Reviewer: ```suggestion
We are currently working on overcoming the above-listed limitations of the simultaneous work of our apps. 
```

---
title: Known issues of AdGuard for Windows
sidebar_position: 10
---

## AdGuard for Windows and AdGuard VPN for Windows compatibility issues

> Once you install AdGuard for Windows and AdGuard VPN for Windows, they just start working together. Such 100% compatibility between the two apps is possible thanks to the unique AdGuard VPN protocol, which allows two VPN apps to work simultaneously on the same device. 

Reviewer 2: тут тоже весьма спорная формулировка - откуда она? дело в том, что компатибилити как минимум на винде зависят в большей степени от реализации, а не от протокола

Author: блин, значит, это я неправильно воспользовалась своими знаниями. Короче, этот заход удаляем.

> However, in some cases there may be problems. If you have both apps enabled and the *Use redirect driver mode* setting is **activated** in AdGuard Ad Blocker, then traffic filtering doesn’t work. The same will happen if the *Filter localhost* setting is **disabled**.  

Reviewer 2: ну вот тут не совсем правильно. Точнее не правильно. тут не `in some cases`,а в 100% случаев. Эти галочки должны быть именно как описано, только в этих условиях будет работать.

Reviewer 2: >The same will happen if the *Filter localhost* setting is **disabled**.

так не получится, ибо бы вырубаем возможность изменения настройки, когда у нас включен ВПН. Либо ты поставь у себя 2 продукта, либо я тебе покажу удалённо - просто лучше 1 раз увидеть

Author: 1. in some cases - тут относится к предыдущему абзацу. типа приложения работают друг с другом идеально. но в некоторых случаях что-то идет не так. например, ...

2. Я видела задизейбленную галку. Но ведь можно отключить впн и поменять настройку адича?

By default, the *Use redirect driver mode* setting is disabled and the *Filter localhost* – is enabled. Usually changing the default state of these settings is required to overcome issues related to the simultaneous operation of AdGuard Ad Blocker and various apps working with the network (e.g. antiviruses, VPNs, network filters, etc).

> The *Use redirect driver mode* setting and the *Filter localhost* setting are located in the Advanced settings section. Mindlessly changing settings there may potentially cause problems with AdGuard's performance, break the Internet connection or compromise your security and privacy. If apps work normally on your device, please do not change Advanced settings states.

> As a temporary solution to the problem with AdGuard filtering in this case, we suggest returning settings to the default state, i.e. disable the *Use redirect driver mode* and enable the *Filter localhost*.

Reviewer 2: не совсем понял, про какую проблему ты говоришь и почему именно "темпоралили солюшен"?

Author: я говорю про проблему с фильтрацией. типа если сменить дефолтные настройки, то перестает работать фильтрация в Адгард. Ну а темпорари - пушто ты ж сказал, что в какой-то версии вы этот момент поправите.
